File: JAVA/01_core_JAVA/Lecture01/isPrime.java
package Lecture01;

import java.util.Scanner;

public class isPrime {
	public static void main(String[] args) {
		Scanner scn = new Scanner(System.in);
		System.out.println("Enter a number to check it is prime");
		int n = scn.nextInt();
		scn.close();
		if (n == 1) {
			System.out.println("1 is neither prime nor composite");
		} else {
			int j = 2;
			while ((j*j) <= n) {
				if (n % j == 0) {
					System.out.println("Composite");
					return;
				}
				j++;
			}
			System.out.println("Prime");
		}
	}

}

File: C++ New/01_Basic/Lec_01/isPrime.cpp
#include<iostream>
using namespace std;
int main(){
    int num;
    cout << "Enter a number to check it is prime or not : " << endl;
    cin >> num;
    if(num == 1){
        cout << "1 is nether prime nor composite" << endl;
        return 0;
    }
    int i = 2;
    while (i * i <= num){
        if (num % i == 0){
            cout << num << " is composite number" << endl;
            return 0;
        }
        i++;
    }
    cout << num << " is a prime number" << endl;
    return 0;
}